core/app/api/customer/group/route.ts

```diff
@@ -0,0 +1,13 @@
+import { NextResponse } from 'next/server';
+
+import { getCustomerGroupId } from '~/client/queries/get-customer';
+
+export async function GET() {
+  try {
+    const { data } = await getCustomerGroupId();
+
+    return NextResponse.json(data);
+  } catch (error) {
+    return NextResponse.json({ error }, { status: 500 });
```

Comment:
I don't know if we want to return the full error here. What details we return depends on the intended use case of the caller. Who/what would be calling this endpoint?